extensions/amp-viewer-integration/0.1/amp-viewer-integration.js

```diff
@@ -160,6 +161,8 @@ export class AmpViewerIntegration {
 
     listenOnce(
       this.win, 'unload', this.handleUnload_.bind(this, messaging));
+
+    new TouchHandler(this.win, messaging);
```

Comment:
Should this go in the `cap` parameter instead of being it's own parameter? I don't care, but if you want to switch to `cap` I'm happy to start using it for anything new.